src/main/java/io/aiven/kafka/connect/common/output/parquet/ParquetSchemaBuilder.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2021 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.parquet;
+
+import java.util.Collection;
+import java.util.Objects;
+
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import io.aiven.kafka.connect.common.config.OutputField;
+
+import io.confluent.connect.avro.AvroData;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/* Build schema for output fields based on SinkRecord values
+* For multiple output fields schema looks like this:
+* {
+*    "type" "record", "fields": [
+*       {"name": "key", type="RecordKeySchema"},
+*       {"name": "offset", type="long"},
+*       {"name": "timestamp", type="long"},
+*       {"name": "headers", type="map"},
+*       {"name": "value", type="RecordValueSchema"},
+*    ]
+* }
+* If record value or record key schema after conversion is type of RECORD,
+* such schema accepts as is, e.g. record value schema:
+* {
+*    "type" "record", "fields": [
+*       {"name": "user", type="string"},
+*       {"name": "address", type="string"},
+*       {"name": "timestamp", type="long"}
+*    ]
+* }*/
```

Comment:
This isn't how it works with other formats. Unwrapping is an orthogonal issue (e.g. see https://github.com/aiven/aiven-kafka-connect-gcs/issues/81) and we probably should not automatically decided it here.

Reply:
Ok I see, it is a bit diff from what we discussed, so solution is like that:
Always use our own Avro schema:
```json
{
    "type": "record", "fields": [
       {"name": "key", type: "RecordKeySchema"},
       {"name": "offset", type: "long"},
       {"name": "timestamp", type: "long"},
       {"name": "headers", type: "map"},
       {"name": "value", type: "RecordValueSchema"},
    ]
}
```
Right?
Because I think it would be useful for customers for such behave in case of Parquet